### ISSUE TITLE
Changing accessibility icon

### ISSUE THREAD

**Issue by Eric69-dev** [joomla/joomla-cms#44775]
### Is your feature request related to a problem? Please describe.
When using "System - Additional Accessibility Features" plugin, the accessibility icon is displayed on front and/or backend site. But there is no way to change this icon.

In fact, the accessibility icon should be this :

![Image](https://github.com/user-attachments/assets/51b1327a-a5f4-4600-a375-a21869ba8322)

and not this one :

![Image](https://github.com/user-attachments/assets/9532f5ee-0968-4630-8932-6c47b9cf59a7)

### Describe the solution you'd like
It should be great to be able to change or override easily the default accessibility icon

**Comment by fgsw** [comment]
@Eric69-dev Please use next time the "Accessibility issue"-template to get automated the "a11y"-label.

**Comment by QuyTon** [comment]
Closing as duplicate of #41191.
